### PR TITLE
Ability to show HTML messages.

### PR DESCRIPTION
**Pull request by mahmoodvcs**
Ability to show HTML message.
Main sweetalert project has this option. I needed it too, so here it is.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -299,7 +317,17 @@
...
         });
       };
-    </script>
+
+      document.querySelector('.sweet-15').onclick = function(){
+		swal({
+		  title: "Sweet HTML!",
+		  text: "Here's a <b>bold</b> and <span style='color:orange;'>colorful</span> message.",
+		  html: true
+		});
```
Comment (lipis) [inline index.html]: fix indentation

File: lib/sweet-alert.js [hunk 2]
```diff
@@ -474,7 +475,10 @@
     $title.innerHTML = escapeHtml(params.title).split("\n").join("<br>");
 
     // Text
-    $text.innerHTML = escapeHtml(params.text || '').split("\n").join("<br>");
+	if(params.html)
+		$text.innerHTML = params.text || '';
+	else
+		$text.innerHTML = escapeHtml(params.text || '').split("\n").join("<br>");
```
Comment (lipis) [inline lib/sweet-alert.js]: indentation, space after `if` and use `{}` even for single line ifs

**Comment by lipis** [comment]
Don't use tabs: https://github.com/lipis/bootstrap-sweetalert/blob/master/.editorconfig#L9

**Comment by iandroogmans** [comment]
HTML support is included in more recent versions of sweetalert. Would be more convenient to upgrade to a newer version of sweetalert. This project is actually a bridge project to support sweetalert in bootstrap. So no changes should be made in sweet-alert itself.

**Comment by lipis** [comment]
As I'm already mentioning it.. in the README.md file.. if anyone is up for a task.. I'm more than happy to review.. https://github.com/lipis/bootstrap-sweetalert/compare/master...t4t5:master

**Comment by mahmoodvcs** [comment]
I fixed indention and braces.

**Comment by lipis** [comment]
Will be solved in #53

**Pull request closed without merging**